testing/btest/language/sizeof.zeek

```diff
@@ -80,6 +90,9 @@ print fmt("Double %s: %f", d, |d|);
 # Size of enum: returns numeric value of enum constant.
 print fmt("Enum %s: %d", ENUM3, |ENUM3|);
 
+# Within a record, enum sizeof should still be ok
+print fmt("Enum in record: %d", |with_enum$e|);
```

Comment:
I'm a bit lost, why we apply `||` here. Isn't the actual `&default` value more interesting? Suggest to add the non-sizeof values, too, minimally to show that they end up producing the same.

```suggestion
print fmt("Enum in record: %d %d", with_enum$e, |with_enum$e|);
```